packages/salesforcedx-vscode-core/src/commands/forceApexClassCreate.ts

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  CliCommandExecutor,
+  Command,
+  SfdxCommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import * as vscode from 'vscode';
+import { channelService } from '../channels';
+import { CancellableStatusBar, taskViewService } from '../statuses';
+import glob = require('glob');
+import { nls } from '../messages';
+import {
+  CancelResponse,
+  CompositeParametersGatherer,
+  ContinueResponse,
+  DirFileNameSelection,
+  ParametersGatherer,
+  SfdxCommandlet,
+  SfdxCommandletExecutor,
+  SfdxWorkspaceChecker
+} from './commands';
+
+class SelectFileName implements ParametersGatherer<{ fileName: string }> {
```

Comment:
I'm almost certain that this `SelectFileName` gatherer should move into the commands.ts file since it would be used by more than one of the wizards. But, you can move it later when you tackle the other wizards.